test/ICSharpCode.SharpZipLib.Tests/Zip/FastZipHandling.cs

```diff
@@ -148,8 +154,23 @@ public void Encryption()
 					ZipEntry entry = zf[0];
 					Assert.AreEqual(tempName1, entry.Name);
 					Assert.AreEqual(1, entry.Size);
-					Assert.IsTrue(zf.TestArchive(true));
+					// Assert.IsTrue(zf.TestArchive(true));
 					Assert.IsTrue(entry.IsCrypted);
+
+					switch (encryptionMethod)
```

Comment:
Codacy found an issue: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745489)